support/cas-server-support-themes-collection/src/main/resources/twbs.properties

```diff
@@ -0,0 +1,16 @@
+cas.theme.name=CAS Boostrap Theme
+cas.theme.description=A boostrap theme
+
+cas.standard.css.file=/webjars/bootstrap/5.0.1/css/bootstrap.min.css,/themes/twbs/css/cas.css,/webjars/font-awesome/5.15.2/css/all.min.css
```

Comment:
This is not good, because it makes it difficult to upgrade versions, and forces manual testing (i.e. if paths ever change, etc). If the theme requires css changes, why not include those files directly in the html in a fragment?